test/benchmark/cli.ts

```diff
@@ -176,32 +176,34 @@ $ node test/benchmark/cli -n 20
             const pjson = await import(
                 pathjoin(process.cwd(), 'packages', packageName, 'package.json')
             );
-            if (pjson.version === '0.0.1') {
+            if (pjson.version === '0.0.1' && opts.compare !== 'none') {
```

Comment:
Double check the logic here to see if there is some way to cover this `0.0.1` logic when adding new features as well as when adding completely new packages.